js/ui/scheduler/ui.scheduler.timeline.js

```diff
@@ -4,9 +4,10 @@ var $ = require("../../core/renderer"),
     noop = require("../../core/utils/common").noop,
     extend = require("../../core/utils/extend").extend,
     registerComponent = require("../../core/component_registrator"),
-    SchedulerWorkSpace = require("./ui.scheduler.work_space"),
+    SchedulerWorkSpace = require("./ui.scheduler.work_space.indicator"),
```

Comment:
It looks like you are requiring an indicator, not workspace. `ui.scheduler.work_space.indicator` seems ambiguous.